lib/copy-sync/__tests__/copy-sync-preserve-time.test.js

```diff
@@ -43,14 +50,14 @@ describeIfPractical('copySync() - preserveTimestamps option', () => {
       const fromStat = fs.statSync(a)
       const toStat = fs.statSync(b)
       if (options.preserveTimestamps) {
-        // https://github.com/nodejs/io.js/issues/2069
-        if (process.platform !== 'win32') {
-          assert.strictEqual(toStat.mtime.getTime(), fromStat.mtime.getTime())
-          assert.strictEqual(toStat.atime.getTime(), fromStat.atime.getTime())
-        } else {
-          assert.strictEqual(toStat.mtime.getTime(), utimes.timeRemoveMillis(fromStat.mtime.getTime()))
-          assert.strictEqual(toStat.atime.getTime(), utimes.timeRemoveMillis(fromStat.atime.getTime()))
-        }
+        // Windows has sub-second support: https://github.com/nodejs/io.js/issues/2069
```

Comment:
I updated the comment after checking the https://github.com/nodejs/io.js/issues/2069 thread - according to its conclusion, we should no longer need to RemoveMillis for Windows (so maybe I should have switched the conditions around instead, or what...? I'm open to reviewing this section)